roles/socle-config/files/crd-conf-dso.yaml

```diff
@@ -1015,6 +1015,11 @@ spec:
                     imageTag:
                       description: Nexus version based on image tag (e.g., "3.56.0").
                       type: string
+                    values:
+                      description: |
+                        WIP: Aims to store nexus values.
```

Comment:
Possible de changer la description sur le modèles des autres description de `values` ? 

Par exemple pour Harbor : 
```yaml
values:
  description: |
    You can merge customs values for harbor, it will be merged with roles/harbor/templates/values/
    See https://github.com/goharbor/harbor-helm
```

Reply:
Il y a un autre changement a effectuer ici à mon avis, puis à répercuter dans le role nexus : nous ne déployons pas Nexus via un chart helm.
C'est la raison pour laquelle il n'a pas de propriété "values" au niveau de la dsc.
Je propose de remplacer ici "values" par une propriété "ingressAnnotations" et de corriger "dsc.nexus['values'].server.ingress.annotations" en "dsc.nexus.ingressAnnotations" dans le fichier "roles/nexus/templates/ingress.yml.j2".